localctlr/LocalController.py

```diff
@@ -347,6 +355,17 @@ def _get_switch_config_in_db(self, switch_name):
         val = d['value']
         return pickle.loads(str(val))
 
+    def _get_switch_internal_config(self, switch_id):
```

Comment:
This function seems to be already added to the LocalController.py with slightly different arg
Would it be possible to check it? 

https://github.com/atlanticwave-sdx/atlanticwave-proto/blob/135b24fb7b5bda3cb5a35a7ce6e20fe87fb82fbf/localctlr/LocalController.py#L369